sktime/utils/numba/stats.py

```diff
@@ -11,9 +11,27 @@ def mean(X):
     return np.mean(X)
 
 
-@njit(fastmath=True, cache=True)
 def median(X):
```

Comment:
Why is there no @njit here?

Reply:
you cannot test X.flags.writeable within njit, 

    @njit(fastmath=True, cache=True)
    def median4(X):
        if X.flags.writeable:
            return _median(X)
        else:
            X = X.copy()
            X.setflags(write=True)
            return _median(X)

File "debug.py", line 145:
    def median4(X):
        if X.flags.writeable:
        ^

During: typing of get attribute at C:\code\sktime-estimator-evaluation\sktime_estimator_evaluation\debug.py (145)

File "debug.py", line 145:
    def median4(X):
        if X.flags.writeable:
        ^